packages/react/lib/react-helmet-shim.js

```diff
@@ -0,0 +1,16 @@
+const deprecate = require('depd')('@untool/react');
+const reactHelmetAsync = require('react-helmet-async');
+
+deprecate(
+  '[DEP004] react-helmet is deprecated. Please switch to react-helmet-async instead (https://github.com/untool/untool/blob/master/DEPRECATIONS.md#dep004).'
+);
+
+module.exports = exports = reactHelmetAsync;
+exports.__esModule = true;
+exports.default = reactHelmetAsync.Helmet;
+
+deprecate.property(
+  module.exports,
+  'default',
+  '[DEP005] The default export of react-helmet-async is deprecated. Please use the named export instead (https://github.com/untool/untool/blob/master/DEPRECATIONS.md#dep005).'
```

Comment:
Hm, I don't understand this depreciation warning.
Isn't this the export for:`import Helmet from 'react-helmet'`? So why isn't this just the same warning as `DEP004`?

Reply:
`react-helmet-async` originally does not provide a default export, while `react-helmet` does.
And we still have a lot of code internally that does `import Helmet from 'react-helmet';` which would then fail. Therefore we provide the default alias ourselves, but at the same time print a deprecation warning so that users switch to using the named export `import { Helmet } from 'react-helmet-async';`.

Reply:
Updated.